test/functional/find_tests.js

```diff
@@ -2733,6 +2733,7 @@ describe('Find', function() {
               cursors[0].next(function(err) {
                 test.equal(null, err);
 
+                cursors[0].close();
```

Comment:
similarly here, are the these cursors not being closed when the client is closed?

Reply:
We don't close cursors when the client is closed.